### PR TITLE
Add project protected tags management

### PR DESCRIPTION
**Pull request by esabouraud**
This PR introduces support for [protected tags](https://docs.gitlab.com/ce/api/protected_tags.html) management.

### CODE REVIEW

**Comment by esabouraud** [comment]
Hi @gpocentek, is there anything wrong with this PR ? Gitlab 11.3 is now released so this new API is available for general use.

**Comment by gpocentek** [comment]
Hi @esabouraud

Sorry for the delay but I don't have a lot of time for reviews these days.

@max-wittig @hans-d do you think you could have a look?

File: docs/gl_objects/projects.rst [hunk 1]
```diff
@@ -657,3 +657,36 @@ Edit project push rules::
...
+Examples
+---------
+
+Get a list of protected tags from a project::
+
+    protected_tags = project.protectedtags.list()
+
+Get a single protected tag or wildcard protected tag::
+
+    protected_tag = project.protectedtags.get('v*')
```
Comment (max-wittig) [inline docs/gl_objects/projects.rst]: This actually doesn't work for me. Just throws 404, even though all my protected tags are called v3.xx and v2.xx

Comment (max-wittig) [reply docs/gl_objects/projects.rst]: Actually it works. GitLab just implemented the protected tag feature in an unexpected way. It actually remembers the way (including wildcards how the tags where protected)

**Comment by max-wittig** [comment]
@hans-d Do you want to take another look or can we merge?

**Comment by hans-d** [comment]
@max-wittig if it's LGTM no need to specifically wait for me unless you have doubts...

**Pull request merged**